engine/src/main/java/org/pentaho/di/trans/step/StepHelperInterface.java

```diff
@@ -0,0 +1,42 @@
+/*! ******************************************************************************
+ *
+ * Pentaho
+ *
+ * Copyright (C) 2024 by Hitachi Vantara, LLC : http://www.pentaho.com
+ *
+ * Use of this software is governed by the Business Source License included
+ * in the LICENSE.TXT file.
+ *
+ * Change Date: 2029-07-20
+ ******************************************************************************/
+
+
+package org.pentaho.di.trans.step;
+
+import org.json.simple.JSONObject;
+import org.pentaho.di.trans.TransMeta;
+
+import java.util.Map;
+
+public interface StepHelperInterface {
+
+  //Common response fields
+  String STATUS = "Status";
+  String ACTION_STATUS = "actionStatus";
+  String SUCCESS_RESPONSE = "Action successful";
+  String FAILURE_RESPONSE = "Action failed";
+  String FAILURE_METHOD_NOT_RESPONSE = "Action failed with method not found";
+
+  //Status codes
+  int SUCCESS_STATUS = 1;
+  int FAILURE_STATUS = -1;
+  int NOT_EXECUTED_STATUS = 0;
```

Comment:
javadoc comment explaining more in depth when these status codes are returned.

-  SUCCESS_STATUS
-  FAILURE_STATUS 
- NOT_EXECUTED_STATUS